config/packages/security.yaml

```
@@ -1,8 +1,11 @@
 parameters:
     sylius.security.admin_regex: "^/admin"
     sylius.security.api_regex: "^/api"
-    sylius.security.new_api_admin_regex: "^/new-api"
     sylius.security.shop_regex: "^/(?!admin|api/.*|api$|media/.*)[^/]++"
+    sylius.security.new_api_route: "/new-api"
+    sylius.security.new_api_admin_route: "/new-api/admin"
+    sylius.security.new_api_admin_regex: "^/new-api/admin"
+    sylius.security.new_api_shop_regex: "^/new-api/(?!admin/.*)[^/]++"
```

```suggestion
    sylius.security.new_api_route: "/new-api"
    sylius.security.new_api_admin_route: "%sylius.security.new_api_route%/admin"
    sylius.security.new_api_shop_route: "%sylius.security.new_api_route%/shop"
    sylius.security.new_api_admin_regex: "^%sylius.security.new_api_admin_route%"
    sylius.security.new_api_shop_regex: "^sylius.security.new_api_shop_route"
```

To be explicit and to not duplicate routing?

---

I did it this way at the beginning, but it didn't look transparent to me, but I will change that 👌